README.md

```diff
@@ -1,33 +1,31 @@
-# [project name]
+# Memory-bias for errors dataset
 
 ## Project Goal
-Replace this text with a pithy description of the goal of the data collection project.
+This dataset will include behavioral and EEG data from the participants with ages between 18-25. Participants exhibit normative variation in social anxiety.
 
 
 ## Background & Design
-Replace this text with a slightly longer (but still only max 250 words) description of the project background and its design. 
+Social anxiety (SA), as one of the most pervasive and long-lasting anxiety disorders [1](https://pubmed.ncbi.nlm.nih.gov/15939837/), is associated with an abnormal enhancement of error monitoring [2](https://pubmed.ncbi.nlm.nih.gov/18271855/). Current theoretical models suggest that enhanced error monitoring is a downstream symptom of SA [3](https://www.frontiersin.org/articles/10.3389/fnhum.2013.00466/full), [4](https://www.frontiersin.org/articles/10.3389/fnhum.2013.00636/full). However, I suggest a novel reconceptualization of the role of error monitoring in SA: error monitoring plays a causal role in the maintenance of SA by orienting attention of individuals towards their errors, improving the encoding and subsequent memory of mistakes, especially within social settings, causing these individuals to exhibit a memory bias for errors and leading to lower self-evaluations of one’s abilities in social settings. As a result of this cascade of neurocognitive processes being repeated across social situations, SA is maintained/worsened. To test this hypothesis, I propose to collect behavioral and EEG data and conduct ERP and time-frequency analyses. I will first determine whether socially anxious individuals exhibit a memory-bias for errors, whether this bias is more prevalent in social settings, and to what degree it is driven by error monitoring. Collectively, this proposal has the potential to reconceptualize how researchers and clinicians think about the role of error monitoring in SA and could inform the design of novel, brain-based interventions.
+
 
 
 ## Roadmap
-:point_right: Details of each data release should be documented in this readme. 
+Release 1: anticipated Q3 2022
 
 
 ## Work in Development
 This `main` branch contains completed releases for this project. For all work-in-progress, please switch over to the `dev` branches.
 
-:point_right: Keep the "Work in Development" text above as the default NDCLab "blurb" to help visitors navigate the repo. But delete this note before publishing the readme file.
+
 
 
 ## Contributors
 | Name | Role |
 | ---  | ---  |
-| insert team member | add team member's role |
-| insert team member | add team member's role |
-| insert team member | add team member's role |
+| Kianoosh Hosseini | Project lead |
+| George Buzzell | guidance |
```

Comment:
I've been working on standardizing this a bit using [these categories](https://www.elsevier.com/authors/policies-and-guidelines/credit-author-statement).  Please flip the two columns and use:
Conceptualization: Kianoosh Hosseini
Project Administration: Kianoosh Hosseini
Supervision: George A. Buzzell